instances.json

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
JSON jest badziewny, ale czytelny. Większość internetów na tym działa i nie tylko. Także jak zapisujesz dane w pliku, to warto z niego korzystać. Jeżeli dojdziesz do granicy jego możliwości, to są lepsze formaty, ale przed tym nie ma co się fatygować

Reply:
To `JSON` ma jakąś przewagę nad `pickle`? Bo jak nie to nie wiem czy jest sens zmieniać funkcje, trochę już pod `pickle` zrobiłem.

Reply:
ma. Właśnie to że się da odczytać. To jest ogromna przewaga, bo nie ogranicza cię do pythona. Kosztem prostoty, bo JSON jest bardzo prosty. Ale w twoim przypadku (i ogólnie, rzekłbym..) to akurat spoko, bo dla ciebie istotne jest tylko to by zapisać i odczytać listę leków z wartościami. Nie potrzebujesz pickle'owe cuda które ci pozwalają zapisywać pythonowe struktury danych.

Inna sprawa jest taka że funkcja która wczytuje/zapisuje do pliku powinna być maksymalnie prosta. Odczyt to otwarcie pliku i zwrócenie zbiór danych, zapis to odwrotnie. Nie bawisz się w usuwaniu pojedynczych elementów, tylko walisz wszystko jak leci do pliku i nara. Dopóki plik ma mniej niż 1MB to nie przejmuj się tym, a to ci bardzo życie ułatwi

Reply:
@mruwnik jeżeli Cię `json` frapguje to spójrz na rozwiązanie `javonet.com` 😄

Reply:
ło panie! To jeszcze bardziej skomplikowane! :D